lib/iris/tests/unit/fileformats/nc_load_rules/actions/__init__.py

```diff
@@ -0,0 +1,157 @@
+# Copyright Iris contributors
+#
+# This file is part of Iris and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+"""
+Unit tests for the module :mod:`iris.fileformats._nc_load_rules.actions`.
+
+This module provides the engine.activate() call used in the function
+`iris.fileformats.netcdf._load_cube`.
+
+"""
+from pathlib import Path
+import shutil
+import subprocess
+import tempfile
+
+import iris.fileformats._nc_load_rules.engine
+from iris.fileformats.cf import CFReader
+import iris.fileformats.netcdf
+from iris.fileformats.netcdf import _load_cube
+
+"""
+Notes on testing method.
+
+IN cf : "def _load_cube(engine, cf, cf_var, filename)"
+WHERE:
+  - engine is a :class:`iris.fileformats._nc_load_rules.engine.Engine`
+  - cf is a CFReader
+  - cf_var is a CFDAtaVariable
```

Comment:
I think CFDAtaVariable should be CFDataVariable here.

Reply:
:+1: